Public/Src/Engine/Processes/ProcessDumper.cs

```diff
@@ -199,7 +199,21 @@ public static bool TryDumpProcessAndChildren(int parentProcessId, string dumpDir
 
                 Process p = maybeProcess.Result;
 
-                if (s_skipProcesses.Contains(p.ProcessName, StringComparer.InvariantCultureIgnoreCase) || p.StartTime > treeDumpInitiateTime)
+                // In NetCore the process name cannot be accessed after the process exits. Noop if the process has exited
+                string processName = null;
+                try
+                {
+                    processName = p.ProcessName;
+                }
+                catch (InvalidOperationException)
+                {
+                    if (!p.HasExited)
+                    {
+                        throw;

```

Comment:
>throw; [](start = 24, length = 6)

Instead of throwing an exception that can crash BuildXL, would it be better to skip the process, but notifies (log or write a dummy file) user that dumping the process failed?